app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -134,7 +141,7 @@ def send_to_vre(user)
       )
     end
 
-    VeteranReadinessEmploymentMailer.build(user, email_addr).deliver_now if user.present?
+    VeteranReadinessEmploymentMailer.build(user, email_addr, @sent_to_cmp).deliver_now if user.present?
```

Comment:
```suggestion
    VeteranReadinessEmploymentMailer.build(user, email_addr, sent_to_cmp).deliver_now if user.present?
```

Reply:
I never needed this to be public in the first place, so I stuck with the ivar.